### PR TITLE
Remove credits

### PR DESCRIPTION
**Pull request by chrissyrissmeyer**
Removes the image credits from examples in changes and history episodes.

### CODE REVIEW

File: _episodes/03-changes.md [hunk 1]
```diff
@@ -298,23 +295,22 @@ $ git commit -m "readme: where are the files?"
 >     From 7248345bacf4593e35c24398a4c375815ee006fc Mon Sep 17 00:00:00 2001
 >     From: Catsy Cline <ccline@pawson.edu>
 >     Date: Mon, 2 Oct 2017 12:43:54 -0700
->     Subject: [PATCH] readme: where are the files?
->
->     ---
->      README.md | 2 ++
->      1 file changed, 2 insertions(+)
+>Subject: [PATCH] readme: where are the files?
```
Comment (dunn) [inline _episodes/03-changes.md]: This needs to be indented at the same level as the rest of the patch.

**Review by dunn — CHANGES_REQUESTED** [review]

File: _episodes/03-changes.md (hunk 1 above)
Comment (chrissyrissmeyer) [reply _episodes/03-changes.md]: fixed, let me know what you think.

**Review by mcritchlow — COMMENTED** [review]
LGTM, but will let @dunn sign off on the requested change.

File: _episodes/03-changes.md [hunk 2]
```diff
@@ -298,23 +295,22 @@ $ git commit -m "readme: where are the files?"
 >     From 7248345bacf4593e35c24398a4c375815ee006fc Mon Sep 17 00:00:00 2001
 >     From: Catsy Cline <ccline@pawson.edu>
 >     Date: Mon, 2 Oct 2017 12:43:54 -0700
->     Subject: [PATCH] readme: where are the files?
+>	  Subject: [PATCH] readme: where are the files?
```
Comment (dunn) [inline _episodes/03-changes.md]: Spaces instead of tabs please 😸

File: _episodes/03-changes.md [hunk 3]
```diff
@@ -298,23 +295,22 @@ $ git commit -m "readme: where are the files?"
...
 >
 >     ---
->      README.md | 2 ++
->      1 file changed, 2 insertions(+)
+>     README.md | 2 ++
```
Comment (dunn) [inline _episodes/03-changes.md]: One more space here and on the next line.

File: _episodes/03-changes.md [hunk 4]
```diff
@@ -298,23 +295,22 @@ $ git commit -m "readme: where are the files?"
...
->      - The black cat, March 1896 (http://ark.digitalcommonwealth.org/ark:/50959/w0892s818). Courtesy Boston Public Library via Digital Commonwealth.
->      - Puzzums in costume (http://photos.lapl.org/carlweb/jsp/DoSearch?index=z&databaseID=968&terms=0000068045). Courtesy Los Angeles Public Library.
->      - A Xmas shopper (http://digitalcollections.nypl.org/items/510d47e1-43b5-a3d9-e040-e00a18064a99). Courtesy New York Public Library.
+>     @@ -3,3 +3,5 @@
+>      The `cats-human-situations.csv` file contains metadata for three image objects.
+>      The original metadata from the source institutions has been abbreviated and made
+>      messier so you have something to clean up!
 >     +
 >     +The images are in the `images/` subdirectory of this repository.
->     --
```
Comment (dunn) [inline _episodes/03-changes.md]: This line needs to stay.

**Review by dunn — APPROVED** [review]

**Pull request merged**